### PR TITLE
[4] Ternary Expression can be safely replaced with condition removing codesmell

### PR DESCRIPTION
**Pull request by PhilETaylor**
Code review

Ternary Expression can be safely replaced with condition removing code smell

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 44ac23fcfe17edc8d9a98e79a1a8bc97dbf5fdd6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31071">issues.joomla.org/tracker/joomla-cms/31071</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 44ac23fcfe17edc8d9a98e79a1a8bc97dbf5fdd6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31071">issues.joomla.org/tracker/joomla-cms/31071</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31071">issues.joomla.org/tracker/joomla-cms/31071</a>.</sub>

File: libraries/src/Mail/Mail.php [hunk 1]
```diff
@@ -714,7 +714,7 @@ public function sendMail($from, $fromName, $recipient, $subject, $body, $mode =
 		}
 
 		// Add sender to replyTo only if no replyTo received
-		$autoReplyTo = (empty($this->ReplyTo)) ? true : false;
+		$autoReplyTo = (empty($this->ReplyTo));
```
Comment (HLeithner) [inline libraries/src/Mail/Mail.php]:
```suggestion
		$autoReplyTo = empty($this->ReplyTo);
```

File: libraries/src/Microdata/Microdata.php [hunk 2]
```diff
@@ -766,7 +766,7 @@ public static function isTypeAvailable($type)
 	{
 		static::loadTypes();
 
-		return (\array_key_exists($type, static::$types)) ? true : false;
+		return (\array_key_exists($type, static::$types));
```
Comment (HLeithner) [inline libraries/src/Microdata/Microdata.php]:
```suggestion
		return \array_key_exists($type, static::$types);
```

**Comment by PhilETaylor** [comment]
Applied suggestions

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**